.travis.yml

```diff
@@ -36,8 +37,10 @@ jobs:
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
     - name: Consumer Tests for s4sdk pipeline (CloudFoundry)
+      if: repo = "SAP/jenkins-library" && ( (type != pull_request && branch =~ /^master$|^it\/.*$/) || (type == pull_request && head_repo = "SAP/jenkins-library" && head_branch =~ /^it\/.*$/) )
       script: cd consumer-test/s4sdk/CloudFoundry && chmod +x runTests.sh && ./runTests.sh
     - name: Consumer Tests for s4sdk pipeline (Neo Environment)
+      if: repo = "SAP/jenkins-library" && ( (type != pull_request && branch =~ /^master$|^it\/.*$/) || (type == pull_request && head_repo = "SAP/jenkins-library" && head_branch =~ /^it\/.*$/) )
```

Comment:
This line is the same line than above. In the long run we should maybe trigger all integration tests at once instead of having idividual sections here in the travis.yml. But I would like to suggest not to adress this here in this PR.